deployment/Rules-Publish-TST-To-PRD.md

```diff
@@ -0,0 +1,48 @@
+# Rules Publish from TST to PRD
+
+1.	Remote desktop into **TDCI01SQL**
+
+2.	Using SQL Server Management Studio connect to the **TDCI01SQL** database server
+
```

Comment:
I'll remove some of these extra lines but the ones tied to an image I'm going to leave them. This way they're separated in a way its obvious which number the image goes to.